### PR TITLE
Split file types for query and analysis

### PR DESCRIPTION
**Pull request by pierrepo**
- File types for query are only associated  (or not) to keywords
- File types for analysis have more informations (MD engine, category, documentation...)

### CODE REVIEW

File: params/query_dev.yml [hunk 1]
```diff
@@ -3,14 +3,21 @@ md_keywords:
...
-- namd
 - md trajectory
 - md trajectories
 - md simulation
 - md simulations
-- amber
 - gromacs
+- gromos
+- namd
+- amber
```
Comment (j0kaso) [inline params/query_dev.yml:13]: Do we really want to have all different amber options in the query_dev?

File: params/query_dev.yml [hunk 2]
```diff
@@ -3,14 +3,21 @@ md_keywords:
...
+- amber96
+- amber99
+- amber14
 - charmm
+- charmm27
+- charmm36
```
Comment (j0kaso) [inline params/query_dev.yml:19]: same with charmm - maybe just charmm + charmm36 is enough?

**Review by j0kaso — COMMENTED** [review]
PR sound in general good to me - I would maybe keep query_dev.yml more slim.

File: params/query_dev.yml (hunk 1 above)
Comment (pierrepo) [reply params/query_dev.yml:13]: Not at all. I will simplify.

File: params/query_dev.yml (hunk 2 above)
Comment (pierrepo) [reply params/query_dev.yml:19]: Yep

**Pull request merged**